nttt/nttt_logging.py

```diff
@@ -0,0 +1,12 @@
+def nttt_display_md(tag_name, text_before, text_after):
+    if text_before != text_after:
+        print("Tag name: {}\n".format(tag_name))
+        print("Replaced: {}{}{}\n".format(tag_name, text_before, tag_name))
+        print("    with: {}{}{}\n".format(tag_name, text_after, tag_name))
+
+def nttt_display_tags(tag_name, text_before, text_after):
+    if text_before != text_after:
+        print("Tag name: <{}>\n".format(tag_name))
```

Comment:
Here's an example of the output:

```
Tag name: **

Replaced: ** Anleitung **

    with: **Anleitung**

```

Are these extra newlines useful? 
Also I'm not sure that having a separate line with tag name is necessary considering the fact that we put effort in formatting "before" and "after" strings in such a way that their comparison (and spotting differences) is trivial.

Reply:
removed